conjureup/controllers/clouds/tui.py

```diff
@@ -18,7 +20,10 @@ def finish(self):
     def render(self):
         utils.info(
             "Summoning {} to {}".format(app.argv.spell, app.provider.cloud))
-        self.finish()
+        if app.provider.cloud_type == cloud_types.LOCALHOST:
+            app.loop.create_task(self._monitor_localhost(self.finish))
```

Comment:
I don't understand why we're doing this task on TUI, since it's non-interactive?  Why not just fail fast if lxd isn't already available?